File: pjecalc-fonte/negocio/br/jus/trt8/pjecalc/negocio/dominio/calculoexterno/ParcelasAtualizaveisUtils.java
/*
 * Decompiled with CFR 0.152.
 */
package br.jus.trt8.pjecalc.negocio.dominio.calculoexterno;

import br.jus.trt8.pjecalc.negocio.comum.MensagemDeRecurso;
import br.jus.trt8.pjecalc.negocio.comum.exceptions.NegocioException;
import java.util.List;

public class ParcelasAtualizaveisUtils {
    protected static void lancarErros(List<MensagemDeRecurso> erros) {
        if (!erros.isEmpty()) {
            NegocioException exception = new NegocioException();
            for (MensagemDeRecurso e : erros) {
                exception.adicionarMensagemDeRecurso(e);
            }
            throw exception;
        }
    }
}

File: src/lib/pjecalc/core/dominio/calculoexterno/parcelas-atualizaveis-utils.ts
/**
 * PJe-Calc v2.15.1 — ParcelasAtualizaveisUtils (base)
 * Porte de: br.jus.trt8.pjecalc.negocio.dominio.calculoexterno.ParcelasAtualizaveisUtils
 */
import { NegocioException } from '../../comum/exceptions/negocio-exception';
import type { MensagemDeRecurso } from '../../comum/mensagem-de-recurso';

export class ParcelasAtualizaveisUtils {
  protected static lancarErros(erros: MensagemDeRecurso[]): void {
    if (erros.length === 0) return;
    const exception = new NegocioException();
    for (const e of erros) {
      exception.adicionarMensagemDeRecurso(e);
    }
    throw exception;
  }
}
